lib/NfcoreSchema.groovy

```diff
@@ -259,7 +259,7 @@ class NfcoreSchema {
         return new_params
     }
 
-    /*
+     /*
```

Comment:
```suggestion
    /*
```

Reply:
Nope, this results in the second linting failure...

![image](https://user-images.githubusercontent.com/17950287/149831612-455dae76-1d4e-4523-8565-091b27ddb231.png)